src/components/Header/index.js

```diff
@@ -0,0 +1,119 @@
+import React, { Fragment } from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import { Trans } from 'react-i18next';
+import { compose } from 'recompose';
+import { injectState } from 'freactal';
+import { withTheme } from 'emotion-theming';
+
+import logoPath from 'theme/images/logo-kids-first-data-portal.svg';
+
+import HouseIcon from 'react-icons/lib/fa/home';
+import DatabaseIcon from 'react-icons/lib/fa/database';
+
+import Dropdown from 'uikit/Dropdown';
+import Row from 'uikit/Row';
+import { uiLogout } from 'components/LogoutButton';
+import { withApi } from 'services/api';
+
+import {
+  NavLink,
+  DropdownLink,
+  HeaderContainer,
+  GradientAccent,
+  HeaderContent,
+  Logo,
+  NavigationGravatar,
+  LinkAsButton,
+  NavBarList,
+  NavbarDropdownWrapper,
+  NavbarDropdownOptionsContainer,
+} from './ui';
+
+const Header = ({
+  state: { loggedInUser },
+  effects: { setUser, setToken, clearIntegrationTokens },
+  theme,
+  history,
+  match: { path },
+  api,
+}) => {
+  const canSeeProtectedRoutes =
+    loggedInUser &&
+    (loggedInUser.roles &&
+      loggedInUser.roles[0] &&
+      loggedInUser.acceptedTerms &&
+      path !== '/join' &&
+      path !== '/');
+  const currentPathName = history.location.pathname;
+  return (
+    <HeaderContainer>
+      <GradientAccent />
+      <HeaderContent>
+        <Row>
+          <Link to="/">
+            <Logo src={logoPath} alt="Kids First Logo" />
+          </Link>
+          {canSeeProtectedRoutes && (
+            <NavBarList ml={40}>
```

Comment:
howcome styled-system style `ml` here? I'd think we should keep things consistent for now & actually introduce styled system if we want this pattern